Ejercicio 13 seba

giunta opened the pull request:
esta es mi version del ejercicio 13 con el main y el de los headers ya editados

giunta committed:
Update main.c

giunta committed:
Update ejercicios.h

giunta committed:
Create ej13.c

gonzafernan commented on ej13.c:
```diff
@@ -0,0 +1,52 @@
+}
+//bucle para calcular el promedio
+float sum=0;
+for (j=1;j<=largo;j++){
```
No es necesario declarar otra variable j, se puede utilizar la misma i del bucle anterior.

gonzafernan commented on ej13.c:
```diff
@@ -0,0 +1,52 @@
+}
+// bucle para ver cual es el numero menor
+
+for (k=2;k<largo;k++){
```
Idem que con j, no hace falta declarar otra variable k, se puede usar la misma i en todos los bucles.

giunta replied:
Seguro? Una ves me que lo trate de hacer me tiro error

gonzafernan commented on ej13.c:
```diff
@@ -0,0 +1,52 @@
+  }
+}
+printf("el vector de numero aleatorios fue: \n");
+for (l=1;l<=largo;l++){
```
Y lo mismo para l jajajajaja

gonzafernan commented on ej13.c:
```diff
@@ -0,0 +1,52 @@
+}
+printf("el vector de numero aleatorios fue: \n");
+for (l=1;l<=largo;l++){
+  printf(" %d, ",vector[l-1]);
```
Corrección: `printf("%ld, ",vector[i-1]);`

gonzafernan commented on ej13.c:
```diff
@@ -0,0 +1,52 @@
+}
+printf("\n");
+printf("el promedio de las componentes del vector fue: %f\n",prom);
+printf("la menor de las componentes del vector fue: %d\n",menor);
```
Corrección: `printf("la menor de las componentes del vector fue: %ld\n",menor);`

gonzafernan committed:
Update ej13.c

Hice un par de cambios, vean si les parece bien. Esta bueno el código del Seba, solo cambie la forma en que encuentra la menor componente (es un poco más sencillo) y hice que el print del vector fuera un poco más bonito jajajjaa
También arregle lo que te comente en el código Seba.

GinoAvanzini committed:
Merge pull request #22 from FernandezGFG/FernandezGFG-patch-ejercicio-13-seba

Update ej13.c
